include/GafferImageUI/ImageGadget.h

```diff
@@ -167,6 +170,15 @@ class GAFFERIMAGEUI_API ImageGadget : public GafferUI::Gadget
 		void setWipeAngle( float angle );
 		float getWipeAngle() const;
 
+
+		void setSelectedIds( std::vector<uint32_t> &&ids );
+
+		void setHighlightId( uint32_t id );
+
+		// Copied from GafferSceneUI/Private/OutputBuffer.h
+		// TODO : Currently public so it can be accessed from TileShaderSelectedIds::ScopedBinding
+		class BufferTexture;
```

Comment:
This feels small enough that I wouldn't mind duplicating it to keep it internal until such a time as IECoreGL provides it.